app/Http/Controllers/ProjectCommentController.php

```diff
@@ -50,7 +50,13 @@ public function edit(ProjectComment $comment)
             return redirect('/')->withErrors('You are not authorized to do this action!');
         }
 
-        return view('project_comments.edit', compact('comment'));
+        $route = 'comments';
+        $object = 'message';
+        $routeToDelete = '/comments/'.$comment->id;
```

Comment:
Is it better to create a variable in the controller instead to create it in the view ?